### PR TITLE
feat(payload): flyPad OS 3.2 - multiple gate boarding when multiple boarding points are opened

### PR DESCRIPTION
**Pull request by 2hwk**
Fixes #[issue_no]

## Summary of Changes

* Adds multiple gate boarding support
- [x] Services Page
- [x] Proper GSX Support (?)
- [x] Estimated boarding time displays correctly
- [x] GSX Config to enable dual gate for GSX

Not addressed in this PR: 
 - EFB Payload Front End
 - Door Animations
 - Ground Vehicles (ask asobo)

## Screenshots (if necessary)

image

FlightSimulator_sUaEUSNy8K

https://github.com/flybywiresim/aircraft/assets/15316958/dcae9b01-8750-4605-a880-b0f8b2984ccb

Note: Outdated, Aft Right door does not work for boarding, only catering. 

https://github.com/flybywiresim/aircraft/assets/15316958/eb5fa3d5-c0aa-4ae4-8b7a-ae1e41244c03

## Additional context

Discord username (if different from GitHub):

## Testing instructions

1. Complete Boarding without any doors/jetway attached, this should work as per the usual boarding process before, then deboard the plane entirely and wait for all sounds to clear (boarding etc.)
2. Go to the services page and open 2 doors (Fwd Left and Aft Left) to enable dual-gate boarding.
3. Go back to the payload page and load passengers (with simbrief or manual entry)
4. The estimated boarding time displayed should be half the duration as previously (less than 10 min) 
5. Start a real or fast time boarding and observe that the estimated duration is accurate, and that the passengers are shown as being filled from both the front and back of the aircraft.
6. Check that deboarding also works in exactly the same way but with the passengers being deboarded from both the front and back of the aircraft.

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Review by tracernz — APPROVED** [review]
JS side looks fine. Didn't review rust. I think we have about 4 different PRs or branches with different A380 package manifests and build scripts now so we might want to just standardise on the comprehensive work @Benjozork and @flogross89 are doing.

File: fbw-a32nx/src/wasm/systems/a320_systems_wasm/src/payload.rs [hunk 1]
```diff
@@ -39,6 +39,22 @@ pub(super) fn payload(builder: &mut MsfsAspectBuilder) -> Result<(), Box<dyn Err
...
+        Variable::aspect("INTERACTIVE_POINT_1_OPEN_REQ"),
+    );
+    builder.copy(
+        Variable::aircraft("INTERACTIVE POINT OPEN", "percent", 2),
+        Variable::aspect("INTERACTIVE_POINT_2_OPEN_REQ"),
+    );
+    builder.copy(
+        Variable::aircraft("INTERACTIVE POINT OPEN", "percent", 3),
+        Variable::aspect("INTERACTIVE_POINT_3_OPEN_REQ"),
+    );
```
Comment (Gurgel100) [inline fbw-a32nx/src/wasm/systems/a320_systems_wasm/src/payload.rs:57]: You actually don't need this. You can add calls to `provides_aircraft_variable` (see lib.rs:290)

File: fbw-a380x/src/wasm/systems/a380_systems_wasm/src/payload.rs [hunk 2]
```diff
@@ -79,6 +79,18 @@ pub(super) fn payload(builder: &mut MsfsAspectBuilder) -> Result<(), Box<dyn Err
...
+        Variable::aspect("INTERACTIVE_POINT_0_OPEN_REQ"),
+    );
+    builder.copy(
+        Variable::aircraft("INTERACTIVE POINT OPEN", "percent", 2),
+        Variable::aspect("INTERACTIVE_POINT_2_OPEN_REQ"),
+    );
+    builder.copy(
+        Variable::aircraft("INTERACTIVE POINT OPEN", "percent", 10),
+        Variable::aspect("INTERACTIVE_POINT_10_OPEN_REQ"),
+    );
```
Comment (Gurgel100) [inline fbw-a380x/src/wasm/systems/a380_systems_wasm/src/payload.rs:93]: You actually don't need this. You can add calls to `provides_aircraft_variable` (see lib.rs:290).

**Review by Gurgel100 — CHANGES_REQUESTED** [review]
My review from the Rust part.

File: fbw-a380x/src/wasm/systems/a380_systems_wasm/src/lib.rs [hunk 3]
```diff
@@ -288,7 +288,9 @@ async fn systems(mut gauge: msfs::Gauge) -> Result<(), Box<dyn Error>> {
     .provides_aircraft_variable("GPS GROUND TRUE TRACK", "Degrees", 0)?
     .provides_aircraft_variable("INDICATED ALTITUDE", "Feet", 0)?
     .provides_aircraft_variable("INTERACTIVE POINT OPEN:0", "Percent", 0)?
-    .provides_aircraft_variable("INTERACTIVE POINT OPEN", "Percent", 3)?
+    .provides_aircraft_variable("INTERACTIVE POINT OPEN", "percent", 2)?
+    .provides_aircraft_variable("INTERACTIVE POINT OPEN", "percent", 3)?
+    .provides_aircraft_variable("INTERACTIVE POINT OPEN", "percent", 10)?
```
Comment (Gurgel100) [inline fbw-a380x/src/wasm/systems/a380_systems_wasm/src/lib.rs:293]: Please capitalize the unit to be consistent.

**Review by Gurgel100 — APPROVED** [review]

**Comment by auroraisluna** [comment]
QA Report - **A32NX**

Discord: alepouna
Object of testing:  #8206
Tier of Testing : 1
Date : 17/09/2023

Testing Process:
- [PASS] Complete Boarding without any doors/jetway attached, this should work as per the usual boarding process before, then deboard the plane entirely and wait for all sounds to clear (boarding etc.)
- Go to the services page and open 2 doors (Fwd Left and Aft Left) to enable dual-gate boarding.
- Go back to the payload page and load passengers (with simbrief or manual entry)
- [PASS] The estimated boarding time displayed should be half the duration as previously (less than 10 min)
- [PASS] Start a real or fast time boarding and observe that the estimated duration is accurate, and that the passengers are shown as being filled from both the front and back of the aircraft.
- [PASS] Check that deboarding also works in exactly the same way but with the passengers being deboarded from both the front and back of the aircraft.

Testing Results:
Passed

**Pull request merged**